### PR TITLE
refactor(autoware_path_distance_calculator): prefix package and namespace with autoware

### PR DESCRIPTION
**Pull request by esteve**
## Description

prefix package and namespace with autoware

## Related links

**Parent Issue:**

- https://github.com/autowarefoundation/autoware/issues/4569

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: common/autoware_path_distance_calculator/src/path_distance_calculator.cpp [hunk 1]
```diff
@@ -54,6 +56,6 @@ PathDistanceCalculator::PathDistanceCalculator(const rclcpp::NodeOptions & optio
     pub_dist_->publish(msg);
   });
 }
-
+}  // namespace autoware::path_distance_calculator
 #include <rclcpp_components/register_node_macro.hpp>
-RCLCPP_COMPONENTS_REGISTER_NODE(PathDistanceCalculator)
+RCLCPP_COMPONENTS_REGISTER_NODE(autowarre::path_distance_calculator::PathDistanceCalculator)
```
Comment (isamu-takagi) [inline common/autoware_path_distance_calculator/src/path_distance_calculator.cpp:61]:
```suggestion
RCLCPP_COMPONENTS_REGISTER_NODE(autoware::path_distance_calculator::PathDistanceCalculator)
```

Comment (esteve) [reply common/autoware_path_distance_calculator/src/path_distance_calculator.cpp]: @isamu-takagi thanks!

**Review by isamu-takagi — APPROVED** [review]
LGTM

File: common/autoware_path_distance_calculator/launch/path_distance_calculator.launch.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <launch>
   <arg name="path" default="/planning/scenario_planning/lane_driving/behavior_planning/path"/>
   <arg name="distance" default="~/distance"/>
-  <node pkg="path_distance_calculator" exec="path_distance_calculator_node" name="path_distance_calculator">
+  <node pkg="autoware_path_distance_calculator" exec="path_distance_calculator_node" name="path_distance_calculator">
```
Comment (a-maumau) [inline common/autoware_path_distance_calculator/launch/path_distance_calculator.launch.xml:4]: I think we need `autoware_` prefix for `exec="path_distance_calculator_node"` too.

**Comment by ktro2828** [comment]
As @a-maumau mentioned, we need to add autowre prefix to the executable too.
I created the PR to fix it could you review that? :pray: https://github.com/autowarefoundation/autoware.universe/pull/8385

**Pull request merged**